script.js

```diff
@@ -96,7 +96,8 @@ function addGenerateCompareUrlButton() {
 		function baseCompareUrlFromCurrentCommitsUrl() {
 			var indexOfCommitsPartOfUrl,
 				githubCommitsUrl,
-				githubCompareUrlLastPart
+				githubCommitsUrlFirstPart,
+				githubCompareUrlLastPart;
```

Comment:
Ouch good catch! This is why its good to have set up linting when you try new code editors :P